couchbase_cluster_admin/cluster.py

```diff
@@ -41,6 +44,25 @@ def __init__(
         self.username = username
         self.password = password
 
+        if connect_through_ssh:
+            if not ssh_username:
+                raise ValueError("You need to specify a `ssh_username`")
+
+            self.ssh_tunnel = SshTunnel(
```

Comment:
This doesn't specify local_bind_address, which seems to default to `('0.0.0.0', RANDOM_PORT)`, possibly exposing the endpoint to other clients in the network. It looks like you can specify `('127.0.0.1',)` to get the automatic port.

Reply:
Modified now to use `('127.0.0.1',)` as you suggested.